openrtb_ext/bidders.go

```diff
@@ -477,6 +478,7 @@ const (
 	BidderSmartHub          BidderName = "smarthub"
 	BidderSmartRTB          BidderName = "smartrtb"
 	BidderSmartyAds         BidderName = "smartyads"
+	BidderSmartx            BidderName = "smartx"
```

Comment:
should add `BidderSmartx `  in alpha sort order

```
BidderSmartx               BidderName = "smartx"
BidderSmartyAds         BidderName = "smartyads"
```

Reply:
@fkoch-sc requesting to address above comment

Reply:
This one's solved.